src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -102,6 +102,41 @@ function convertAssigneesToSelectorItems(approvers: Array<TaskCollabAssignee>):
     });
 }
 
+function getEditErrorMessage(taskType: TaskType, editMode?: TaskEditMode, error?: { status: number }): React.Node {
```

Comment:
This should ideally be its own component `<TaskError>`

Reply:
yah could do. at first this was simply to get the message object, but there were changes to both messages used. can separate this out.